components/forms/LoginForm/LoginForm.tsx

```diff
@@ -39,15 +39,13 @@ const LoginForm = () => {
 				onChange={(e) => setEmail(e.target.value)}
 				type="email"
 				label="Email"
-				isError={invalidEmail ? true : false}
-			/>
+				isError={invalidEmail ? true : false} placeholder={""}			/>
```

Comment:
```suggestion
				isError={invalidEmail} 
				placeholder={""}/>
```

jaki invalidEmail ma type? wydaje mi się, że brakuje w ogóle typowania tutaj, tak czy inaczej pradopodobnie to boolean, więc nie potrzeba `true ? false` a jeśli nie, to na pewno da się to zapisać jako `!!invalidEmail`

+ masz jakieś złe wcięcia tutaj z placeholderem